conjure-codegen/src/objects.rs

```diff
@@ -28,7 +28,7 @@ pub fn generate(ctx: &Context, def: &ObjectDefinition) -> TokenStream {
 
     quote! {
         use conjure_object::serde::{ser, de};
-        use conjure_object::serde::ser::SerializeMap as SerializeMap_;
+        use conjure_object::serde::ser::SerializeStruct as SerializeStruct_;
```

Comment:
Do we have tests that check that the ser/de still round-trips with the serialization changes?

Reply:
Yeah, we have tests in conjure-test that assert a specific JSON output.